src/modules/0.0.19/iasql_functions/index.ts

```diff
@@ -8,5 +8,9 @@ class IasqlFunctions extends ModuleBase {
     super.init();
   }
   iasqlOperationType = IasqlOperationType;
+  // TODO: here for testing purpose. To be delete it
+  customCall = async () => {
```

Comment:
How are parameters passed in here?

Reply:
I found out below, as a single `string[]` argument. It might be better to `...` it (as suggested below) for more natural looking functions.

Reply:
Also, can you confirm that this form also works?

```suggestion
  async customCall() {
```

It *should*, but I'd prefer confirmation since that's the only way to get access to `this` in these custom functions.

Reply:
I can test it, I also think it should work, but let you know. Also, I expect this to change once the `rpc` subtype is in place so was not so thoughtful on the method definition.

Reply:
Right. We need to think about that. The reasoning behind it was to prevent users from manually calling an RPC on a function not meant to be an RPC, but it will probably need to be instantiated like the `extends BaseMapper` classes in order to have access to the module's `this` property and be able to do non-trivial things.

Reply:
It worked